test/e2e/sql/src/test/resources/cases/dql/dql-integration-select-system-schema-mysql.xml

```diff
@@ -156,6 +156,10 @@
     <test-case sql="SELECT * FROM information_schema.key_column_usage limit 1" db-types="MySQL" scenario-types="tbl" adapters="proxy">
         <assertion expected-data-source-name="read_dataset" />
     </test-case>
+
+    <test-case sql="SELECT * FROM information_schema.keywords limit 1" db-types="MySQL" scenario-types="tbl" adapters="proxy">
+        <assertion expected-data-source-name="read_dataset" />
+    </test-case>
```

Comment:
- @FlyingZC Is there any background issue for #27905? I don't quite understand why the table cannot be found in https://github.com/apache/shardingsphere/actions/runs/7772894056/job/21195958057?pr=29982#step:7:3426.
```shell
Error:  2024-02-04 09:09:24.319 [main] o.a.s.t.e2e.framework.E2EExtension - Error case: proxy: tbl -> MySQL -> Literal -> SELECT * FROM information_schema.keywords limit 1, message: java.sql.SQLSyntaxErrorException: Unknown table 'keywords' in information_schema
```

Reply:
- Modified the default MySQL Server version in `org.apache.shardingsphere.test.e2e.env.container.atomic.storage.impl.MySQLContainer` to 8.0 to solve the problem.